macOSUpgrade.sh

```diff
@@ -212,7 +212,7 @@ fvStatus=$( /usr/bin/fdesetup status | head -1 )
 
 ##Check if device is on battery or ac power
 pwrAdapter=$( /usr/bin/pmset -g ps )
-if [[ ${pwrAdapter} == *"AC Power"* ]]; then
+if [[ "$pwrAdapter" == *"AC Power"* ]]; then
     pwrStatus="OK"
```

Comment:
```
$ /usr/bin/pmset -g ps
Now drawing from 'AC Power'
 -InternalBattery-0 (id=3866723)	100%; charged; 0:00 remaining present: true
$
```

How about like this?
```
$/usr/bin/pmset -g ps | head -1 | tr -d \' | awk '{print $(NF -1)}'
AC
$
```

```
pwrAdapter=$( /usr/bin/pmset -g ps  | /usr/bin/head -1 | /usr/bin/tr -d \' | /usr/bin/awk '{print $(NF -1)}' )
if [  "$pwrAdapter" = AC ]; then
   pwrStatus="OK"
```

Reply:
I changed this. https://github.com/kc9wwh/macOSUpgrade/pull/114/commits/ee4d58cdf8e7b100c272db4a869c5b558bbf4cec